scene.go

```diff
@@ -11,14 +11,14 @@ var scenes = make(map[string]*sceneWrapper)
 // Scene represents a screen ingame.
 // i.e.: main menu, settings, but also the game itself
 type Scene interface {
-	// Preload is called before loading resources
-	Preload()
```

Comment:
Yes. I changed my mind. I was trying to simplify but running against myself when I talk about flexibility. This method gives flexibility to the developer.